### ISSUE TITLE
Question- rollup Angular2 *library*

### ISSUE THREAD

**Issue by daniele-zurico** [rollup/rollup#1125]
Hello everyone I'm a newbie of rollup and I'm trying to understand if what I want to achieve is possible with rollup.
My scenario is a library that has inside different base components (so no bootstrap application but just a list of component that I want to use in my main application).
The structure is pretty much like that:
index.js - export all the modules
folderA
  |-- moduleA
  |-- componentA
  |-- html
  | -- scss
folderB
  |-- moduleB
  |-- componentB
  |-- html
  | -- scss

The first operation that I did is ngc (AOT)
1)Make sense now to create a dist folder generated by rollup that contains a bundle.js?
2) If make sense how works to import the moduleA inside my main application?
```
import {moduleA} from 'my-library';
@NgModule({
    imports: [
        *moduleA*
]})
```
3)Is possible/make sense to create different bundle for different components?

**Comment by Rich-Harris** [comment]
I'm afraid I don't understand the question, could you rephrase it please? Note that Rollup is an entirely separate project from Angular, you might be better off asking for help in an Angular forum, or on Stack Overflow.

**Comment by daniele-zurico** [comment]
Thanks for the reply I think that I already have the reply. My question was if is possible to generate multiple output per each folder, instead of a single bundle.

**Comment by Rich-Harris** [comment]
No, you'd need a separate config for each output.
